components/page/navbar/index.js

```diff
@@ -0,0 +1,200 @@
+import React from "react";
```

Comment:
Make the name as `Navbar/index.js`, plus it shouldn't be in the `pages` directory, but outside it as it is not the page component.